e2e-polybft/e2ehelper/execute_bridging_config.go

```diff
@@ -48,6 +48,9 @@ type RestartValidatorsConfig struct {
 	WaitTime   time.Duration
 	StartIndxs []int
 	StopIndxs  []int
+
+	StopBlade  bool
```

Comment:
Maybe it would be better to have a single `type ExecutableType int` enum with values `Oracle = 0, Blade = 1, BladeAndOracle = 2`, because this way, we are changing the current behavior of all tests (nothing will be stopped).